feg/gateway/services/session_proxy/credit_control/gx/definitions.go

```diff
@@ -27,6 +27,14 @@ const (
 	RuleLevel    MonitoringLevel = 0x1
 )
 
+type MonitoringSupport uint8
```

Comment:
I think we should consolidate the types near line 23 and place consts into line 25. just a nit though :)

Reply:
Let's move it there.

To me it was more visual this way so we can now what AVPs results are implemented. But we can check the types

Reply:
I just checked on `gy/definitions.go`. There we have more constants and they are organized by message type. I think for that case it is easier to see the codes that belong to each AVP.   

|So I prefer to leave it this way instead of consolidating. This way we can have a clear definition of the AVPs.